gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy

```diff
@@ -51,6 +51,8 @@ class LaunchConfigTask extends DefaultTask {
 
     static final List<String> dirs = ['var/data/tmp']
 
+    static final Map<String, String> defaultEnvironment = Collections.unmodifiableMap(['MALLOC_ARENA_MAX':'4'])
```

Comment:
Might be worth including a comment referencing https://issues.apache.org/jira/browse/HADOOP-7154 for some of the background on why this is being set